main.cpp

```diff
@@ -8,38 +8,37 @@ int main() {
     Student student2{"Anna", "Kowalska", "Katowice", "wesoła", "1", 4321, "98876543210", Gender::Female};
     Student student3{"Justyna", "Dunka", "Poznan", "wesoła", "1", 4321, "98876543210", Gender::Female};
 
-     Database db;
+    Database db;
     db.printAll();
     db.addStudent(student);
     db.addStudent(student2);
     db.addStudent(student3);
 
     db.printAll();
 
-    std::cout<<"---------SORT BY PESEL-----------\n";
+    std::cout << "---------SORT BY PESEL-----------\n";
     db.sortByPesel();
     db.printAll();
 
-    std::cout<<"---------SORT BY SURENAME-----------\n";
-    db.sortbySurename();
+    std::cout << "---------SORT BY SURENAME-----------\n";
```

Comment:
My suggest:
std::cout << "---------SORT BY SURNAME-----------\n";

Reply:
It's typo. Maybe should all uses of word surname be renamed to lastname. It's more official.
I see need to fix  2 typo in sortbySurename.